src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -55,9 +67,46 @@ public double getAverageScore() {
    * @param topNComments the number of highest magnitudes to retrieve
    */
   public Statistics(List<UserComment> userCommentList, int topNComments) {
+    wordsToIgnore = CommonWordsRetriever.getCommonWords();
```

Comment:
Why do we need a separate class for it? I think we can directly create one here?

Reply:
Spoke offline - next step is getting the list of words to ignore from a text file, which will all be done in the CommonWordsRetriever class